core/domain/rights_manager.py

```diff
@@ -579,11 +579,8 @@ def check_can_manage_voice_artist_in_activity(user, activity_rights):
     """
     if activity_rights is None:
         return False
-    elif (role_services.ACTION_CAN_MANAGE_VOICE_ARTIST in user.actions and (
-            activity_rights.community_owned or activity_rights.is_published())):
-        return True
-    else:
-        return False
+
+    return role_services.ACTION_CAN_MANAGE_VOICE_ARTIST in user.actions
```

Comment:
Maybe add a comment explaining why this *doesn't* require the activity to be public. 

(We shouldn't allow *adding* voiceover artists to private exps, I think?)

Although I think you have addressed this below, you'll probably want something in the docstring here to say "Callers are expected to ensure that ..." Also please make sure there are comprehensive tests for the different scenarios.

Reply:
Done.